OWNERS_ALIASES

```diff
@@ -358,12 +358,12 @@ aliases:
     - neolit123
     - vincepri
   sig-network-api-approvers:
-    - smarterclayton
+    - danwinship
```

Comment:
I think API approvers need to go thru API review "training" (shadowing, mostly).  Happy to do that, of course.